src/App.jsx

```diff
@@ -1,58 +1,107 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
 
+import cn from 'classnames';
+import { useState } from 'react';
+
+const SORT_FIELD_ALPHABET = 'name';
+const SORT_FIELD_LENGTH = 'length';
```

Comment:
```suggestion
const SORT_FIELD = {
   ALPHABET: 'name',
   LENGTH: 'length',
   NONE: 'none',
 }
```
consider to make one constant as object for all sort fields. and better to make for none sort instead empty string